src/components/Common/CloseButton/__snapshots__/CloseButton.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CloseButton should be rendered 1`] = `ShallowWrapper {}`;
```

Comment:
As you can see such snapshot does not make any sense `ShallowWrapper {}`
You can try to use `expect(closeButton.html()).toMatchSnapshot();` or `render` instead of `shallow`

Same for ModalDialog.test.jsx
StyledButton.test.jsx
etc